generators/app/templates/static/scss/utilities/_utilities.typography.scss

```diff
@@ -11,23 +13,23 @@
 
     font-family: $font-secondary;
     line-height: 107%;
-    letter-spacing: -1.33 / 64 * 1em;
+    letter-spacing: math.div(-1.33, 64) * 1em;
```

Comment:
ovo trebamo vidjet da se pojednostavi mozda; nije tu bas potrebna neka napredna matematika. figma nam daje vrijednost u postotku i to mozemo prepisat u em-ove. npr. -1% iz figme je 0.01em u nasem CSS-u... nek ostane, al samo komentar da smo/sam to overengineerali